packages/blob/src/index.ts

```diff
@@ -1,7 +1,9 @@
+type BlobPart = BufferSource | Blob | string;
+
 /**
  * @type {Record<string, File|undefined>}
  */
-const cache = {};
+const cache: Record< string, File | undefined > = {};
```

Comment:
I think the cache type should be `Record< string, File >`. We never intentionally store `undefined` values.

Reply:
Also, we can remove all of the JSDoc type annotations.